package.json

```diff
@@ -37,6 +38,7 @@
     "@nestjs/cli": "^9.0.0",
     "@nestjs/schematics": "^9.0.0",
     "@nestjs/testing": "^9.0.0",
+    "@types/babel__core": "^7.1.19",
```

Comment:
do you need this?

Reply:
aveam erori legate de babel_core, am instalat pachetul ca sa scap de erori